book/extend-admin.rst

```diff
@@ -0,0 +1,822 @@
+Extend Admin UI
+===============
+
+The final section of the book has a deeper look at how to introduce your own navigation items, lists, forms and field
```

Comment:
`has a deeper look at how to integrate your own entities and views into the Sulu administration interface. This includes introducing new navigation entities, configuring new list views and configuring new form views.`

Reply:
would omit field types